ROMFS/px4fmu_common/mixers/BigFat.aux.mix

```diff
@@ -0,0 +1,19 @@
+V-shape Vertical Tail 1 mixer(Right)
+--------------
```

Comment:
The main output didn't work fine with the main.mix, so i want to use the aux output to control the servos.